specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2024-05-15-preview/networkSecurityPerimeter.json

```diff
@@ -0,0 +1,455 @@
+    },
+    "ProvisioningIssue": {
+      "description": "Describes provisioning issue for given NetworkSecurityPerimeterConfiguration",
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "name": {
+          "description": "Name of the issue",
+          "type": "string"
+        },
+        "properties": {
+          "description": "Properties of provisioning issue",
+          "type": "object",
+          "readOnly": true,
+          "properties": {
+            "issueType": {
+              "description": "Type of issue",
+              "enum": [
+                "Unknown",
+                "ConfigurationPropagationFailure"
+              ],
+              "type": "string",
+              "x-ms-enum": {
+                "name": "IssueType",
+                "modelAsString": true
+              }
+            },
+            "severity": {
+              "description": "Severity of the issue.",
+              "enum": [
+                "Warning",
+                "Error"
+              ],
+              "type": "string",
+              "x-ms-enum": {
+                "name": "Severity",
+                "modelAsString": true
+              }
+            },
+            "description": {
+              "description": "Description of the issue",
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "NspAccessRule": {
+      "description": "Information of Access Rule in Network Security Perimeter profile",
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "name": {
+          "description": "Name of the resource",
+          "type": "string"
+        },
+        "properties": {
+          "description": "Properties of Access Rule",
+          "readOnly": true,
+          "type": "object",
+          "properties": {
+            "direction": {
+              "description": "Direction of Access Rule",
+              "enum": [
+                "Inbound",
+                "Outbound"
+              ],
+              "type": "string",
+              "x-ms-enum": {
+                "name": "NspAccessRuleDirection",
+                "modelAsString": true
+              }
+            },
+            "addressPrefixes": {
+              "description": "Address prefixes in the CIDR format for inbound rules",
+              "type": "array",
+              "items": {
+                "description": "IP Address",
+                "type": "string"
+              }
+            },
+            "subscriptions": {
+              "description": "Subscriptions for inbound rules",
+              "type": "array",
+              "items": {
+                "description": "Subscription for inbound rule",
+                "type": "object",
+                "readOnly": true,
+                "properties": {
+                  "id": {
+                    "description": "The ARM identifier of subscription",
+                    "type": "string"
+                  }
```

Comment:
refer common types v5 param ? SubscriptionIdParameter

Reply:
1. Since this is a not a parameter, doesn't make sense to include that.
2. Referring to a similar approved and merged PR https://github.com/Azure/azure-rest-api-specs/pull/27804 , these changes weren't suggested there. Keeping this the same to ensure consistency.

Reply:
sounds right, AFAIR these are ARM identifiers, not just subscripton ids.